opflexagent/test/test_endpoint_file_manager.py

```diff
@@ -1204,3 +1209,23 @@ def test_dhcp_ep_svi(self):
 
     def test_dhcp_ep_no_svi(self):
         self._test_dhcp_ep()
+
+    def test_snat_to_fip(self):
```

Comment:
'''Mapping between host snat ips to the floating ips'''

Reply:
Done.